src/seedu/addressbook/commands/CommandResult.java

```diff
@@ -27,6 +27,14 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     }
 
     /**
+     *  Returns feedback message for the command
+     */
+    
+    public String getFeedbackToUser() {
+		return feedbackToUser;
+	}
+
+	/**
```

Comment:
Please fix the indentation and change the tab to spaces in Eclipse, see [here](http://stackoverflow.com/questions/407929/how-do-i-change-eclipse-to-use-spaces-instead-of-tabs)